includes/sanitizers/class-amp-core-theme-sanitizer.php

```diff
@@ -1932,6 +1998,257 @@ public function add_twentytwenty_toggles() {
+
+				wp_add_inline_style( 'twenty-twenty-one-style', $new_styles );
+			},
+			11
+		);
+	}
+
+	/**
+	 * Amend the Twenty Twenty-One stylesheet to make it compatible with the changes made to the document during
+	 * sanitization.
+	 */
+	public static function amend_twentytwentyone_styles() {
+		add_action(
+			'wp_enqueue_scripts',
+			static function() {
+				$style_handle = 'twenty-twenty-one-style';
+
+				// Bail if the stylesheet is not enqueued.
+				if ( ! wp_style_is( $style_handle ) ) {
+					return;
+				}
+
+				$css_file = get_theme_file_path(
+					sprintf( 'style%s.css', is_rtl() ? '-rtl' : '' )
+				);
+
+				if ( ! file_exists( $css_file ) ) {
+					return;
+				}
+
+				// Set the registered handle as an alias for other stylesheets to depend on.
+				wp_styles()->registered[ $style_handle ]->src = false;
+
+				$styles = file_get_contents( $css_file ); //phpcs:ignore WordPress.WP.AlternativeFunctions.file_get_contents_file_get_contents
+
+				// Make the rules for the mobile menu less specific so that it can also be applied to the
+				// amp-lightbox mobile menu also.
+				$new_styles = str_replace( '.primary-navigation > .primary-menu-container', '.primary-navigation .primary-menu-container', $styles );
+				$new_styles = str_replace( '.primary-navigation > div > .menu-wrapper', '.primary-navigation div > .menu-wrapper', $new_styles );
+
+				// Append any extra rules that may be needed.
+				$new_styles .= '
+					.primary-navigation-open .menu-button-container {
+						/* Needs to be shown above amp-lightbox, which has a z-index of 1000. */
+						z-index: 1001;
+					}
+
+					@media only screen and (max-width: 481px) {
+						/* Hide the desktop menu so that it does not interfere when the amo-lightbox mobile menu is open. */
+						.primary-navigation > .primary-menu-container {
+							display: none;
+						}
+
+						/* Accommodate for the admin bar height */
+						.admin-bar amp-lightbox .primary-menu-container {
+							margin-top: var(--global--admin-bar--height);
+						}
+					}
+
+					/* Show the sub-menu on hover of menu item */
+					.primary-navigation div > .menu-wrapper .menu-item-has-children:hover > .sub-menu {
+						display: block;
+					}
+
+					/* Hide the plus icon on hover of menu item */
+					.primary-navigation div > .menu-wrapper .menu-item-has-children:hover > .sub-menu-toggle > .icon-plus {
+						display: none;
+					}
+
+					/* Show the minus icon on hover of menu item */
+					.primary-navigation div > .menu-wrapper .menu-item-has-children:hover > .sub-menu-toggle > .icon-minus {
+						display: flex;
+					}
+
+					/* In the Twenty Twenty-One theme, when on a non-AMP page and the mobile menu is open, the switcher
+					is incorrectly shown at the top. In that case the switcher is hidden. */
+					body.lock-scrolling > #amp-mobile-version-switcher {
+						display: none;
+					}
+				';
+
+				wp_add_inline_style( $style_handle, $new_styles );
+			},
+			11
```

Comment:
@pierlon Setting this to `9` fixes my issue here: https://github.com/ampproject/amp-wp/pull/5648/files#r538826941

Reply:
This didn't work for me, as at priority 9 the `twenty-twenty-one-style` script is not registered yet. It gets enqueued and registered in `twenty_twenty_one_scripts()` which happens at priority 10.